README.md

```diff
@@ -23,9 +23,33 @@ Launch the application from the executable `LungAIR-SuperBuild/Slicer-build/Lung
 
 ## Using the MONAI Deploy docker image
 
-_(to be written)_
+By default, the LungAir application runs the underlying MONAI Torch model with "Locally saved model weights, without MONAI deploy".  To instead use "MONAI Deploy with locally saved model weights" or "MONAI Deploy with docker image", select that option for the "Backend model" in the Advanced section in the lower left of the main screen.  If you have not yet created a docker image to use, you will need to do that first, with commands similar to:
+
+```shell
+# Useful directory and paths
+cd lungair-desktop-application
+DEPLOY_APP='Modules/Scripted/Home/HomeLib/deploy_app.py'
+MODEL_PATH='Modules/Scripted/Home/Resources/PyTorchModels/LungSegmentation/model0018.pth'
+DOCKER_BASE='nvcr.io/nvidia/pytorch:22.09-py3'
+NEW_DOCKER='ghcr.io/kitwaremedical/lungair-desktop-application/lung_air_model_deploy:latest'
```

Comment:
The idea is that `NEW_DOCKER` can refer to whatever docker registry the user wants, right?

Right now I think it's not totally clear to the reader that they are supposed to figure out for themselves what to put for `NEW_DOCKER`.

Can we somehow rephrase things so that the specific url `ghcr.io/kitwaremedical/...`  is just never mentioned? Instead, can we describe explicitly how the user should decide what to put for their own `NEW_DOCKER`?

Reply:
Addressed.

Reply:
It looks good now! Ready to merge.